internal/hmyapi/blockchain.go

```diff
@@ -72,9 +72,5 @@ func (s *PublicBlockChainAPI) GetStorageAt(ctx context.Context, address common.A
 // given block number. The rpc.LatestBlockNumber and rpc.PendingBlockNumber meta
 // block numbers are also allowed.
 func (s *PublicBlockChainAPI) GetBalance(ctx context.Context, address common.Address, blockNr rpc.BlockNumber) (*hexutil.Big, error) {
-	state, _, err := s.b.StateAndHeaderByNumber(ctx, blockNr)
-	if state == nil || err != nil {
-		return nil, err
-	}
-	return (*hexutil.Big)(state.GetBalance(address)), state.Error()
+	return s.b.GetBalance(address)
```

Comment:
Why you just ignored the blockNr? That's an important input parameter.

Reply:
Have you tested the API of the old code and new code? What are the behaviors?

Reply:
agree. right now for testing we will try to get the latest balance first. We will add complete behaviour later. Let me add todo

Reply:
I suspect the old code works.